samples/dml.js

```diff
@@ -288,6 +288,21 @@ function updateUsingDmlWithStruct(instanceId, databaseId, projectId) {
         params: {
           name: nameStruct,
         },
+        types: {
```

Comment:
I am not able to recall, with this change it is mandatory to pass type for dml queries ?

Reply:
Only for struct. not for any other